### ISSUE TITLE
determine if 'rawDataVersion' annotation is being used

### ISSUE THREAD

**Issue by kdaily** [MEP-LINCS/MEP_LINCS#91]
Not sure where this came from - it's on the current schema, but no values set on data.

**Comment by markdane** [comment]
It's used in PreprocessMEMACell.R when getting raw data form a local server

**Comment by markdane** [comment]
rawDataVersion is used in PreprocessMEMACell.R when getting data from a local server. I don't think it's used as an annotation but I think it should stay as we are likely to have newer versions of raw data that are based on the same images.

**Comment by kdaily** [comment]
That's what I meant - if it should be on as an annotation anywhere. If not, I'll remove from schema.

**Comment by markdane** [comment]
rawDataVersion should remain and all of the raw data files should have values of v2 for the current releases

**Comment by kdaily** [comment]
Is it correct that there is no raw data (level 0) in the pre-release folders? Need to figure this out for this release.

**Comment by markdane** [comment]
You're correct. This release reprocesses the raw data from the prior release.

**Comment by kdaily** [comment]
OK, we should make that clear in the release notes (if it isn't noted already!). It will be difficult (and confusing, I think) to re-list all of those files. However, I'm confused about setting `v2` as the `rawDataVersion` - is this the value it had at the time of the previous release?

**Comment by kdaily** [comment]
To be clear, all of the files in this query should have `rawDataVersion = 'v2'`?

https://goo.gl/F4oJt6

**Comment by markdane** [comment]
yes, that's right

**Comment by kdaily** [comment]
Some of these already have `Segmentation = 'v2'` - is this the same thing?

**Comment by markdane** [comment]
Yes and that's confusing. The segmentation pipeline creates the raw data. Internally, we organize the data into a directory named by the version of the pipeline. So segmentation pipeline version v2 creates raw data that is at version v2.

It looks to me that rawDataVersion should be a command line option in the pipeline but not an annotation on any of the files.

**Comment by kdaily** [comment]
I think that having both annotations with the same values is confusing. Also, the released table (https://www.synapse.org/#!Synapse:syn8313677) doesn't have those annotations in the table, so we would need to add them.

**Comment by kdaily** [comment]
OK! I just noted that the release notes for the last release say v2 was used. but there are raw files with `Segmentation=v2.1` and `Segmentation=v1` in the released data. I think that with our new file view system we should rethink the organization - just want to make sure it is correct though!

**Comment by kdaily** [comment]
Not going to use `rawDataVersion`. Mark will roll back all level 0 files with `v2.1` to `v2` and double check on `v1` files - if they should be sequestered and not made public.

**Comment by kdaily** [comment]
What's the status here @markdane?

**Comment by markdane** [comment]
I don't have time now to roll this back but it doesn't need to hold up our release.

**Comment by kdaily** [comment]
What did we decide on this @markdane?
